examples/evaluation/promptiter/server/client.py

```
@@ -0,0 +1,317 @@
+        help="Disable parallel evaluation across eval cases.",
+    )
+    return parser.parse_args()
+
+
+def request_json(method: str, url: str, payload: dict | None = None) -> dict:
+    data = None
+    headers = {"Accept": "application/json"}
+    if payload is not None:
+        data = json.dumps(payload).encode("utf-8")
+        headers["Content-Type"] = "application/json"
+    request = urllib.request.Request(url, data=data, headers=headers, method=method)
```

Ruff (0.15.10)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)